### PR TITLE
[10.0][ADD] mrp_flattened_bom_xlsx

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
# MRP Flattened BOM Report XLSX

This module extends the functionality of the MRP capabilities of Odoo, and allows you to export the Flattened BOM to MS Excel .XLSX format. 

Flattened BOM remove intermediate process between last product and necessary products to produce. It's a good tool to summarize required components. Use it under your responsibility.

### CODE REVIEW

File: mrp_flattened_bom_xlsx/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+MRP Flattened BOM Report XLSX
+=====================================
+
+This module extends the functionality of the MRP capabilities of Odoo,
+and allows you to export the flattened BOM to MS Excel .XLSX format.
+It also mantain units correctly.
```
Comment (JordiBForgeFlow) [inline mrp_flattened_bom_xlsx/README.rst]: Please be more precise. Something like: it shows the quantities of lowest level components in the base unit of measure of the product.

File: mrp_flattened_bom_xlsx/README.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+MRP Flattened BOM Report XLSX
+=====================================
+
+This module extends the functionality of the MRP capabilities of Odoo,
+and allows you to export the flattened BOM to MS Excel .XLSX format.
```
Comment (JordiBForgeFlow) [inline mrp_flattened_bom_xlsx/README.rst:10]: Please describe what is a flattened bom

File: mrp_flattened_bom_xlsx/report/flattened_bom_xlsx.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                        totals[line.product_id] += line.product_uom_id.\
+                            _compute_quantity(
+                            line.product_qty*factor, line.product_id.uom_id)
+                    else:
+                        totals[line.product_id] = line.product_uom_id.\
+                            _compute_quantity(
+                            line.product_qty*factor, line.product_id.uom_id)
+
+        workbook.set_properties({
+            'comments': 'Created with Python and XlsxWriter from Odoo 9.0'})
```
Comment (JordiBForgeFlow) [inline mrp_flattened_bom_xlsx/report/flattened_bom_xlsx.py]: 10.0

File: mrp_flattened_bom_xlsx/static/description/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Export BOM Structure to Excel Totals</h2>
```
Comment (JordiBForgeFlow) [inline mrp_flattened_bom_xlsx/static/description/index.html]: Please align to README

**Comment by HviorForgeFlow** [comment]
cc @jbeficent

**Review by JordiBForgeFlow — APPROVED** [review]
Please squash your commits

**Review by LoisRForgeFlow — APPROVED** [review]
Looks great! :+1:

**Pull request merged**